docs/content/Caching/Using-Pre-Aggregations.md

```diff
@@ -485,3 +356,79 @@ services:
 [ref-config-extdriverfactory]: /config#options-reference-external-driver-factory
 [ref-production-checklist-refresh]:
   /deployment/production-checklist#set-up-refresh-worker
+
+## Pre-Aggregations Storage
+
+The default pre-aggregations storage in Cube.js is its own purpose-built storage layer: Cube Store.
+
+Alternatively, you can store pre-aggregations either **internally**, in the source database, or **externally**,  in databases, such as MySQL or Postgres.
+
+In order to make external pre-aggregations work outside of Cube Store, you should set the
+[`externalDriverFactory`][ref-config-extdriverfactory] and
+[`externalDbType`][ref-config-extdbtype] properties in your `cube.js`
+configuration file. These properties can also be set through the environment
+variables.
+
+```bash
+CUBEJS_EXT_DB_HOST=<YOUR_DB_HOST_HERE>
+CUBEJS_EXT_DB_PORT=<YOUR_DB_PORT_HERE>
+CUBEJS_EXT_DB_NAME=<YOUR_DB_NAME_HERE>
+CUBEJS_EXT_DB_USER=<YOUR_DB_USER_HERE>
+CUBEJS_EXT_DB_PASS=<YOUR_DB_PASS_HERE>
+CUBEJS_EXT_DB_TYPE=<SUPPORTED_DB_TYPE_HERE>
+```
+
+<!-- prettier-ignore-start -->
+[[warning |]]
+| Please be aware of the limitations when using internal and external (outside of Cube Store) pre-aggregations.
+<!-- prettier-ignore-end -->
+
+![](https://raw.githubusercontent.com/cube-js/cube.js/master/docs/content/Caching/pre-aggregations.png)
+
+#### Some known limitations of using Postgres/MySQL as a storage layer listed below.
```

Comment:
```suggestion
#### Some known limitations when using Postgres/MySQL as a storage layer listed below.
```

Reply:
How about:

```suggestion
### Known limitations
#### External pre-aggregations
```

Reply:
@hassankhan I'm to make it hierarchical but don't want to make it show in the sidebar.